papers/index.php

```diff
@@ -126,23 +106,14 @@
 <UL>
 <?php
 print_papers(array("sc-2008",
-                   "iccs-2008",
-                   "euro-pvmmpi-2008-xrc"));
+	           ));
 ?>
 </UL>
 
 <p><strong>2007</strong></p>
 <UL>
 <?php
-print_papers(array("euro-pvmmpi-2007-mtt",
```

Comment:
The MTT paper still gets cited - but we should note it on the MTT repo. I'll add a PR to add it to the README.md at root.

Reply:
Added a PR here: https://github.com/open-mpi/mtt/pull/640